omnibus/files/private-chef-cookbooks/private-chef/recipes/nginx.rb

```diff
@@ -243,7 +243,11 @@
 end
 
 # Set up runit service for nginx component
-component_runit_service 'nginx'
+component_runit_service 'nginx' do
+  action [:enable, :start]
```

Comment:
are both the enable and start actions required here?

Reply:
Yes Prajakta, start is not working if i do multiple restores due to some missing directories. enable and start are solving these issues